src/index.spec.ts

```diff
@@ -60,6 +60,8 @@ describe('Client', function () {
   let dbAddr: string
   describe('.newDB', () => {
     it('response should succeed', async () => {
+      const identity = await Libp2pCryptoIdentity.fromRandom()
+      await client.getToken(identity)
```

Comment:
This is the only test we have right now for getToken... we should add a proper test for it.